docs/combine.rst

```diff
@@ -0,0 +1,105 @@
+combine
+=======
+
+.. dfhack-tool::
+    :summary: Combine stacks of food and plant from one or every stockpile.
+    :tags: fort productivity stockpiles items plants
+
+
+Usage
+-----
+
+::
+
+    combine [help] [<mode opts>] [<stockpile opts>] [<stack opts>] [<output opts>]
+
+
+Examples
+--------
+``combine``
+    Displays help
+``combine --preview --stockpile=all``
+    Preview stack changes for all types in all stockpiles.
+``combine --preview --stockpile=all --max=500``
+    Preview stacks changes with max of 500 per stack for all stockpiles.
+``combine --preview --types=meat,plant``
+    Preview meat and plant stacks changes in all stockpiles.
+``combine --merge --stockpile=all --max=500``
+    Merge stacks for all stockpiles with max of 500 per stack.
+``combine --merge --stockpile=here``
+    Merge stacks for stockpile under cursor; enable cursor through DF settings.
+``combine --merge --stockpile=<num> --types=meat,plant``
+    Merge meat stacks for stockpile id <num>.
+``combine --info=stockpiles.txt``
+    Preview stack changes and write detailed info to stockpiles.txt.
```

Comment:
what use case is this intended for? would it be more appropriate as a separate `devel/` tool?

Reply:
I can take out this option. I used it for testing.

Reply:
I used this for testing, so can get rid of this option.